x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/add_prepackaged_rules_schema.test.ts

```diff
@@ -1117,6 +1117,61 @@ describe('add prepackaged rules schema', () => {
         language: 'kuery',
         version: 1,
         timeline_id: 'timeline-id',
+        timeline_title: 'timeline-title',
+      }).error
+    ).toBeFalsy();
+  });
+
+  test('You cannot omit timeline_title when timeline_id are present', () => {
+    expect(
+      addPrepackagedRulesSchema.validate<
+        Partial<Omit<RuleAlertParamsRest, 'meta'> & { meta: string }>
```

Comment:
copypasta fix:

You only need this:
```ts
addPrepackagedRulesSchema.validate<Partial<RuleAlertParamsRest>>({
```

Since you are not messing around with 'meta'